src/main/resources/templates/Landing.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+  <title>Online HTML Editor</title>
+  <link rel="stylesheet" href="mystyle.css">
+

```

Comment:
Title and link tags should be moved within the head tag below

Reply:
We'll discuss with Nolan, I only modified some of the HTML for Thymeleaf stuff